dfx.json

```diff
@@ -16,6 +16,15 @@
       "wasm": "./dummy-issuer/dummy_issuer.wasm.gz",
       "build": "./dummy-issuer/build.sh",
       "shrink": false
+    },
+    "internet_identity": {
+      "type": "custom",
+      "wasm": "https://github.com/dfinity/internet-identity/releases/download/release-2024-04-05/internet_identity_dev.wasm.gz",
+      "candid": "https://github.com/dfinity/internet-identity/releases/download/release-2024-04-05/internet_identity.did",
+      "url": {
```

Comment:
What do these URLs do? Should one be pointing to the dev environment?

Reply:
I removed them. I copied it from another project but I believe they are not necessary.